Integrations/MISP_V2/test_constants.py

```diff
@@ -0,0 +1,328 @@
+full_response_before_filtering = [
```

Comment:
We shouldn’t have another py file in the integration directory, if you want to add files that are relevant to the tests add a directory called test_data and put those files in there